src/Common/src/Common/Util/MimeTypeUtils.cs

```diff
@@ -17,6 +17,7 @@ public static class MimeTypeUtils
         public static readonly MimeType APPLICATION_JSON = new MimeType("application", "json");
```

Comment:
Should we be using at least some of `System.Net.Mime.MediaTypeNames` here or instead?

Reply:
Ya .. let me take a look and see what the impacts would be

Reply:
Oh .. I looked at that a long time ago.. and I didn't see much to use...  but might be missing something .. do you have specific ideas?

Reply:
I don't have very specific ideas and don't know that I fully understand what we're doing in/with this class, just wanted to make sure we're not needlessly duplicating. It also looks like this code isn't new, so I'm fine with not changing anything here if you don't think there's anything to gain by changing it

Reply:
Ok.. lets leave it as is